docs/faststore/docs/managing-performance/managing-lazy-loading-sections-on-plp.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: "Managing lazy loading sections on Product Listing Pages (PLPs)"
+---
+
+By default, FastStore improves performance by wrapping store sections in `LazyLoadingSection` and ViewportObserver` components. These components delay loading non-essential sections until they are needed, such as when a user scrolls near them. This approach reduces initial page load times and improves user experience, particularly on slower networks or devices.
+
+While lazy loading improves performance, it can harm SEO. Search engine crawlers often do not execute JavaScript or wait for dynamically loaded content. As a result, critical sections like product grids or category pages may not be indexed. For instance, if a user searches for "gym clothes," your product listings might not appear in search results if they are not included in the initial HTML.
+
+In this guide, learn how to optimize SEO by controlling lazy loading for sections in FastStore.
+
+## Before you begin
+
+Make sure the `@faststore/cli` package is updated to the latest version. If you need to update it, follow the instructions on [Updating the '@fastore/cli' package version](https://developers.vtex.com/docs/guides/faststore/project-structure-updating-the-cli-package-version)
+
+## Instructions
+
+### Step 1: Configuring SEO for Product Listing Pages (PLP)
+
+Configure the title and description templates for product listing pages (PLP) in the discovery.config.js file to optimize your store's SEO and allow dynamically generating metadata based on the collection name.
```

Comment:
suggestion: since the title template would also affect the other types of PLPs, I believe it would be good to not talk only about "collection" -- but I'm not sure on how to change this text hahah what do you think? Also, if the category/brand doesn't have a description configured on Catalog, the description template would be used. 
```suggestion
Configure the title and description templates for product listing pages (PLP) in the `discovery.config.js` file to optimize your store's SEO and allow dynamically generating metadata based on the collection/category/subcategory/brand name.
```